float64[] goal_vals
bool cartesian_goal
---
bool success
---
